addons/im_livechat/static/src/embed/common/livechat_button.js

```diff
@@ -1,59 +1,22 @@
-import { SESSION_STATE } from "@im_livechat/embed/common/livechat_service";
-import { Component, useExternalListener, useRef, useState } from "@odoo/owl";
-
-import { useMovable } from "@mail/utils/common/hooks";
+import { Component, useRef, useState } from "@odoo/owl";
 
 import { useService } from "@web/core/utils/hooks";
 import { debounce } from "@web/core/utils/timing";
 
-const LIVECHAT_BUTTON_SIZE = 56;
-
 export class LivechatButton extends Component {
     static template = "im_livechat.LivechatButton";
     static props = {};
     static DEBOUNCE_DELAY = 500;
 
     setup() {
-        this.store = useService("mail.store");
+        this.store = useState(useService("mail.store"));
```

Comment:
`useState()` around `useService()` is no longer necessary in 18.1 (`useService()` automatically `useState()` when the service is stateful, i.e. it returns a reactive)